Code/Tools/SceneAPI/SDKWrapper/SceneWrapper.h

```diff
@@ -31,8 +31,31 @@ namespace AZ
 
             virtual void Clear();
 
+            enum class AxisVector
+            {
+                X = 0,
+                Y = 1,
+                Z = 2,
+                Unknown
+            };
+
+            virtual AZStd::pair<AxisVector, int32_t> GetUpVectorAndSign() const;
+            virtual AZStd::pair<AxisVector, int32_t> GetFrontVectorAndSign() const;
+            // some SceneAPI might force the usage of custom root transformation
+            // O3DE shall not try to reorient scene in such cases
+            virtual AZStd::optional<SceneAPI::DataTypes::MatrixType> UseForcedRootTransform() const;
+            virtual float GetUnitSizeInMeters() const;
+            virtual AZ::Aabb GetAABB() const;
+            virtual uint32_t GetVerticesCount() const;
```

Comment:
This this SceneWrapper.h kind of like the "api" everyone inherits from to implement a new wrapper?  If so, we should heavily comment these virtuals, using the standard doxygen slash-slash-exclaim way
```cpp
//! Sentence ending in a period that serves as the short description.
//! Additional sentences describing when to use this or other information not obvious from
//! the name of the function, such as what the default return would be if not overridden.
```